### PR TITLE
[New] How to Install Apps on Kubernetes with Helm 3

### PR DESCRIPTION
**Pull request by hzoppetti**
New Helm 3 Guide
Updated Helm 2 with links to v2 Helm docs

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit b5298eeb43054b33e73ebd8c538e979eef15f4d6

https://deploy-preview-2957--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit b2e7666e45d4d7df74e3e070b34a02f153ddd963

https://deploy-preview-2957--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Request for when this is review/merged:

If the Octant PR #2898 is already merged, please update the Helm link on line 101 in that guide so that it points to the Helm 2-specific guide from this current PR.

**Comment by rsyracuse** [comment]
Add a deprecation notice per this PR https://github.com/linode/docs/pull/3020

File: docs/kubernetes-shortguide-definitions/helm-tiller-definition-shortguide/index.md [hunk 1]
```diff
@@ -15,4 +15,4 @@ show_on_rss_feed: false
 
 ## Helm Tiller
 
-A server component that runs on your cluster and receives commands from the Helm client software. [Tiller](https://helm.sh/docs/glossary/#tiller) is responsible for directly interacting with the Kubernetes API (which the client software does not do). Tiller maintains the state for your Helm releases.
+A server component that runs on your cluster and receives commands from the Helm client software before it's deprecation in [Helm 3](https://helm.sh/docs/faq/#removal-of-tiller). [Tiller](https://v2.helm.sh/docs/glossary/#tiller) is responsible for directly interacting with the Kubernetes API (which the client software does not do). Tiller maintains the state for your Helm releases.
```
Comment (andystevensname) [inline docs/kubernetes-shortguide-definitions/helm-tiller-definition-shortguide/index.md]: This reads a little weird. Maybe just a second sentence that states: "Tiller has been deprecated in Helm 3."

Comment (hzoppetti) [reply docs/kubernetes-shortguide-definitions/helm-tiller-definition-shortguide/index.md]: added sentence

**Comment by hzoppetti** [comment]
rebased to grab the k8s alpha deprecation short guide

**Pull request merged**